### DISCUSSION TITLE
post/typescript-exercises-with-gpt/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#6] [Comments]
# post/typescript-exercises-with-gpt/

GPT를 활용하면서 Typescript Exercises를 푼 과정을 정리합니다.

https://jgjgill.com/post/typescript-exercises-with-gpt/

**Comment by colorkite10** [comment]
이렇게 정리된 글을 읽으니 확실히 TypeScript 스터디 때에 잘 이해가 안 가던 부분이 와닿네요! 물론 아직 제가 한번에 습득하기엔 무리가 있지만요 :)
